1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Unit tests
+subtitle: Developer guide
+---
+
+<link rel="import" href="../../../0.5/components/google-youtube/google-youtube.html">
+
+{% include toc.html %}
+
+Use Web Component Tester (`wct`) to unit test your Polymer elements. 
+
+`wct` is an end-to-end testing environment built by the Polymer core team. It
+enables you to test your elements locally, against all of your installed
+browsers, or remotely, via Sauce Labs. `wct` is composed of the following
+tools:
+
+*   [Mocha](https://mochajs.org/) for test framework, complete with support 
+    for BDD and TDD.
+*   [Chai](http://chaijs.com/) for assertions. 
+*   [Sinon](http://sinonjs.org/) for spies, stubs, and mocks.
+*   [Selenium](http://www.seleniumhq.org/) for running tests against 
+    multiple browsers. 
+
+## Quick start
+
+Watch the Polycast below to learn how to set up `wct` and create and run
+some simple unit tests.
+
+<div class="yt-embed">
+  <google-youtube
+    videoid="YBNBr9ECXLo"
+    autoplay="0"
+    rel="0"
+    fluid>
+  </google-youtube>
+</div>
+
+If you prefer step-by-step instructions, the instructions below 
```

Comment:
I don't think "Go watch this Polycast" makes sense as the first sentence of a quick start section. I think the order's backwards. 

Suggest a flow like:

This section shows you how to:

- Set up WCT
- Create a unit test
- Run the unit test

Follow the steps below to get set up, or watch the Polycast:

[polycast embed]